src/main/java/bio/terra/janitor/service/cleanup/flight/GoogleBigQueryTableCleanupFlight.java

```diff
@@ -0,0 +1,25 @@
+package bio.terra.janitor.service.cleanup.flight;
+
+import bio.terra.cloudres.common.ClientConfig;
+import bio.terra.janitor.db.JanitorDao;
+import bio.terra.stairway.Flight;
+import bio.terra.stairway.FlightMap;
+import bio.terra.stairway.RetryRuleFixedInterval;
+import org.springframework.context.ApplicationContext;
+
+/** A Flight to cleanup BigQuery Table in GCP. */
+public class GoogleBigQueryTableCleanupFlight extends Flight {
+  public GoogleBigQueryTableCleanupFlight(FlightMap inputParameters, Object applicationContext) {
+    super(inputParameters, applicationContext);
+    JanitorDao janitorDao =
+        ((ApplicationContext) applicationContext).getBean("janitorDao", JanitorDao.class);
+    ClientConfig clientConfig =
+        ((ApplicationContext) applicationContext).getBean("crlClientConfig", ClientConfig.class);
```

Comment:
Also renamed GoogleDatasetCleanupFlight(Step) to GoogleBigQueryDatasetCleanupFlight(Step).

Reply:
(1) We can not use autowire in Stairway :(
(2) I think we need to have the name "crl" because that is the clientConfig for using CRL. If we just have clientConfig, I feel it is confusing "which client" if Janitor needs SAM client config, RBS client config? 
(3) Created a constant class to hold bean name.

Reply:
Thanks!